backend/open_webui/models/message_metrics.py

```
@@ -0,0 +1,658 @@
+                query = db.query(MessageMetric)
+                if domain:
+                    query = query.filter(MessageMetric.user_domain == domain)
+                if model:
+                    query = query.filter(MessageMetric.model == model)
+                return query.count()
+        except Exception as e:
+            logger.error(f"Failed to get messages number: {e}")
+            return 0
+
+    def get_daily_messages_number(
+        self, days: int = 1, domain: Optional[str] = None, model: Optional[str] = None
```

Ruff (0.12.2)

Unused method argument: `days`

(ARG002)